scripts/time.js

```diff
@@ -0,0 +1,13 @@
+/*global $ setInterval moment serverTimezone*/
```

Comment:
As-is this code is intimately tied to the PM page so I would prefer it to be next to the page that uses it (`tools/project_manager/projectmgr.js`).

Reply:
Done this.